### PR TITLE
Update doc for feature Swing Mode added to Modbus integration

### PR DESCRIPTION
**Pull request by crug80**
## Proposed change

Updated documentation ofr Swing mode feature added to ModBus Integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113710
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by janiversen** [comment]
You miss the link to the PR in core.

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -682,6 +682,40 @@ climates:
         value 1 is written."
       required: false
       type: integer
+    swing_mode_register:
+      description: "Configuration of the register for Swing mode"
+      required: false
+      type: map
+      keys:
+        address:
+          description: "Address of Swing mode register. (int to call write_register, list of 1 int to call write_registers)"
```
Comment (janiversen) [inline source/_integrations/modbus.markdown]: The register is a fixed type?

I does not seem documented which type it is.

**Review by janiversen — CHANGES_REQUESTED** [review]
A question.

**Review by janiversen — CHANGES_REQUESTED** [review]
missing information.

File: source/_integrations/modbus.markdown (hunk 1 above)
Comment (crug80) [reply source/_integrations/modbus.markdown]: 2 lines after that at line 693 you can see ` type: [integer, list]`

Comment (janiversen) [reply source/_integrations/modbus.markdown]: No which request are used to read the register.

Comment (crug80) [reply source/_integrations/modbus.markdown]: > No which request are used to read the register.

Uhm.. Actually the integration uses "holding" also for fan mode, hvac mode and, obviously in this case also for swing mode without a clear declaration of that.  
Do you mean that I should clarify this into the documentation and the same for the others?!

**Review by janiversen — CHANGES_REQUESTED** [review]
Please solve the CI errors.

**Review by janiversen — DISMISSED** [review]
LGTM, thanks.

File: source/_integrations/modbus.markdown [hunk 2]
```diff
@@ -682,6 +682,40 @@ climates:
         value 1 is written."
       required: false
       type: integer
+    swing_mode_register:
+      description: "Configuration of the register for Swing mode"
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
      description: "Configuration of the register for swing mode"
```

File: source/_integrations/modbus.markdown [hunk 3]
```diff
@@ -682,6 +682,40 @@ climates:
...
+      required: false
+      type: map
+      keys:
+        address:
+          description: "Address of Swing mode register. (int to call write_register, list of 1 int to call write_registers). - Reading done through holding register"
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
          description: "Address of swing mode register. (int to call write_register, list of 1 int to call write_registers). - Reading done through holding register"
```

File: source/_integrations/modbus.markdown [hunk 4]
```diff
@@ -682,6 +682,40 @@ climates:
...
+          required: true
+          type: [integer, list]
+        values:
+          description: "Mapping between the register values and Swing modes"
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
          description: "Mapping between the register values and swing modes"
```

File: source/_integrations/modbus.markdown [hunk 5]
```diff
@@ -682,6 +682,40 @@ climates:
...
+          required: true
+          type: map
+          keys:
+            swing_mode_state_on:
+              description: "Value corresponding to Swing mode On."
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
              description: "Value corresponding to swing mode on."
```

File: source/_integrations/modbus.markdown [hunk 6]
```diff
@@ -682,6 +682,40 @@ climates:
...
+              required: false
+              type: integer
+            swing_mode_state_off:
+              description: "Value corresponding to Swing mode Off."
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
              description: "Value corresponding to swing mode off."
```

File: source/_integrations/modbus.markdown [hunk 7]
```diff
@@ -682,6 +682,40 @@ climates:
...
+              required: false
+              type: integer
+            swing_mode_state_horizontal:
+              description: "Value corresponding to Swing mode Horizontal."
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
              description: "Value corresponding to swing mode horizontal."
```

File: source/_integrations/modbus.markdown [hunk 8]
```diff
@@ -682,6 +682,40 @@ climates:
...
+              required: false
+              type: integer
+            swing_mode_state_vertical:
+              description: "Value corresponding to Swing mode Vertical."
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
              description: "Value corresponding to swing mode vertical."
```

File: source/_integrations/modbus.markdown [hunk 9]
```diff
@@ -682,6 +682,40 @@ climates:
...
+              required: false
+              type: integer
+            swing_mode_state_both:
+              description: "Value corresponding to Swing mode Both."
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
              description: "Value corresponding to Swing mode both."
```

**Review by frenck — APPROVED** [review]
Thanks, @crug80 👍 

../Frenck

**Pull request merged**